### ISSUE TITLE
HW Review

### ISSUE THREAD

**Issue by faisalrcb** [AllenJeffreyGurfel/Portfolio#2]
Great work! Nice job with html structure and indentation. Also noticed that you are trying to use JavaScript, awesome. And its nice to see your implemented Google fonts right away. 

One thing to note, I didn't see a heroku link for your submission. Its not big deal for this assignment, but make sure you are good to go with that.

**Comment by AllenJeffreyGurfel** [comment]
Yeah, I submitted the assignment before we covered Heroku. Here it is: https://rocky-cove-92403.herokuapp.com/about.html
